### PR TITLE
Dispute readability

### PR DESCRIPTION
**Pull request by arkanoider**
@grunch 

playing a bit with some refactoring insipired a bit by `cursorAI`, he did quite well in `App.rs` imo and I tried to chunck big functions in `dispute.rs` to start.

Take a look with a grain of saly, just compiled not tested.

## Summary by CodeRabbit

- **New Features**
	- Introduced new functions for improved message handling and dispute event publishing.
	- Added enhanced logging for error handling and action processing.
- **Bug Fixes**
	- Updated message creation for disputes and actions to improve clarity and maintainability.
- **Refactor**
	- Streamlined action handling processes, reducing redundancy and improving code organization.
	- Improved error handling and logging mechanisms across various functions.

### CODE REVIEW

File: src/app.rs [hunk 1]
```diff
@@ -35,13 +35,62 @@
...
+        Action::AdminCancel => admin_cancel_action(msg, event, my_keys, pool, ln_client).await,
+        Action::AdminSettle => admin_settle_action(msg, event, my_keys, pool, ln_client).await,
+        Action::AdminAddSolver => admin_add_solver_action(msg, event, my_keys, pool).await,
+        Action::AdminTakeDispute => admin_take_dispute_action(msg, event, pool).await,
+
+        _ => {
+            tracing::info!("Received message with action {:?}", action);
+            Ok(())
+        }
+    }
```
[the remark this answers is not in the document]
Comment (arkanoider) [reply src/app.rs:83]: Will add later

File: src/app/dispute.rs [hunk 2]
```diff
@@ -33,36 +160,12 @@ pub async fn dispute_action(
 
     // Check dispute for this order id is yet present.
     if find_dispute_by_order_id(pool, order_id).await.is_ok() {
-        error!("Dispute yet opened for this order id: {order_id}");
+        tracing::error!("Dispute yet opened for this order id: {order_id}");
         return Ok(());
     }
```
[the remark this answers is not in the document]
Comment (arkanoider) [reply src/app/dispute.rs:165]: Removed `tracing::error` message and adding your suggestion.

File: src/app.rs [hunk 3]
```diff
@@ -35,13 +35,62 @@ use nostr_sdk::prelude::*;
...
+        // Order-related actions
+        Action::NewOrder => order_action(msg, event, my_keys, pool).await,
+        Action::TakeSell => take_sell_action(msg, event, my_keys, pool).await,
+        Action::TakeBuy => take_buy_action(msg, event, my_keys, pool).await,
+
+        // Payment-related actions
+        Action::FiatSent => fiat_sent_action(msg, event, my_keys, pool).await,
+        Action::Release => release_action(msg, event, my_keys, pool, ln_client).await,
+        Action::AddInvoice => add_invoice_action(msg, event, my_keys, pool).await,
+        Action::PayInvoice => todo!(),
```
[the remark this answers is not in the document]
Comment (arkanoider) [reply src/app.rs:64]: Will do later

**Comment by grunch** [comment]
Hey buddy tests are failing

**Comment by arkanoider** [comment]
Yep! Too many rabbit advices...will fix!

**Comment by arkanoider** [comment]
let's see now...

**Review by grunch — APPROVED** [review]
tACK

**Pull request merged**